### ISSUE TITLE
SEACAS: SuplibCpp/Ioss: error w/ nvcc on vortex

### ISSUE THREAD

**Issue by jewatkins** [trilinos/Trilinos#6552]
## Bug Report
@trilinos/seacas

### Description
I get the following error while building w/ the latest Trilinos develop + SEACAS=ON on vortex:
```
.../Trilinos/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h: In function ‘fmt::v6::internal::fp fmt::v6::internal::normalize(fmt::v6::internal::fp)’:
.../Trilinos/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h:378:43: error: ‘significand_type’ was not declared in this scope
       static const int significand_size = bits<uint64_t>::value;
                                           ^~~~~~~~~~~~~~~~
.../Trilinos/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h:378:43: note: suggested alternative: ‘significand_size’
       static const int significand_size = bits<uint64_t>::value;
                                           ^~~~~~~~~~~~~~~~
                                           significand_size
.../Trilinos/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h:378:59: error: template argument 1 is invalid
       static const int significand_size = bits<uint64_t>::value;
                                                           ^
.../Trilinos/packages/seacas/libraries/suplib_cpp/fmt/core.h: In member function ‘void fmt::v6::internal::arg_map<Context>::push_back(fmt::v6::internal::value<Context>) [with Context = fmt::v6::basic_format_context<std::back_insert_iterator<fmt::v6::internal::buffer<char> >, char>]’:
.../Trilinos/packages/seacas/libraries/suplib_cpp/fmt/core.h:1079:6: note: the ABI of passing aggregates with 16-byte alignment has changed in GCC 5
     void push_back(value<Context> val)
      ^~~~~~~~~
```
I didn't see an issue in the code so I'm not sure why the compiler doesn't like it. It looks like there was an attempt to fix this w/ https://github.com/trilinos/Trilinos/commit/417e2ebef09041f8ccd6effb9c9f4141ce0302c9 . I got past it by replacing `significand_type` but that's probably not the cleanest fix:
```
diff --git a/packages/seacas/libraries/ioss/src/fmt/format-inl.h b/packages/seacas/libraries/ioss/src/fmt/format-inl.h
index bd6c6bfdde..fb0a94f0c3 100644
--- a/packages/seacas/libraries/ioss/src/fmt/format-inl.h
+++ b/packages/seacas/libraries/ioss/src/fmt/format-inl.h
@@ -360,7 +360,7 @@ namespace internal {
     significand_type f;
     int              e;
 
-    static FMT_CONSTEXPR_DECL const int significand_size = bits<significand_type>::value;
+    static FMT_CONSTEXPR_DECL const int significand_size = bits<uint64_t>::value;
 
     fp() : f(0), e(0) {}
     fp(uint64_t f_val, int e_val) : f(f_val), e(e_val) {}
diff --git a/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h b/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h
index bd6c6bfdde..fb0a94f0c3 100644
--- a/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h
+++ b/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h
@@ -360,7 +360,7 @@ namespace internal {
     significand_type f;
     int              e;
 
-    static FMT_CONSTEXPR_DECL const int significand_size = bits<significand_type>::value;
+    static FMT_CONSTEXPR_DECL const int significand_size = bits<uint64_t>::value;
 
     fp() : f(0), e(0) {}
     fp(uint64_t f_val, int e_val) : f(f_val), e(e_val) {}
```

### Steps to Reproduce
1. SHA1:  1c59af4fdbfe99ed74fa569d3c96958868e4e84a
1. Configure script:  Can provide if needed.
1. Configure log:  Can provide if needed.
1. Build log:  Can provide if needed.
1. compilers/tpls:
gcc/7.2.1
spectrum-mpi/2019.06.24
cuda/10.1.243
boost-1.72.0
hdf5-1.10.6
lapack-3.9.0
netcdf-c-4.7.3
pnetcdf-1.12.1
zlib-1.2.11

**Comment by mhoemmen** [comment]
@trilinos/seacas @gsjaardema

**Comment by gsjaardema** [comment]
Will have a PR up soon.  I don't see any reason why `significand_type` can't just be replaced by `unint64_t` in the four places it is used...

**Comment by jewatkins** [comment]
Okay great, thanks!

**Comment by jewatkins** [comment]
Just confirmed this was fixed by #6558, thanks!
